src/keyword_vpd_parser.cpp

```diff
@@ -133,4 +133,91 @@ void KeywordVpdParser::checkNextBytesValidity(uint8_t i_numberOfBytes)
     }
 }
 
+types::DbusVariantType KeywordVpdParser::readKeywordFromHardware(
```

Comment:
can we make it simple,
use populateVpdMap() which gets the keyword map for us.
use this map to get the particular keyword value.

In this way we no need to implement reading logic here .

Reply:
that won't be simple when there is a read keyword request from D-bus. as for each and every request, populateVpdMap() creates the complete map of keyword-value pairs which is not required in this case.

Reply:
It depends on the where the keywond, it can be found at the beginning or can be found at the end.

Reply:
I don't need a map with all keyword values for my case. so why should I use populateVpdMap()